vip-filesystem.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Plugin Name:       VIP Filesystem Plugin
+ * Description:       Provides a custom stream wrapper for handling file uploads to VIP/A8C Files API.
+ * Version:           1.0.0
+ * Author:            Automattic
+ * Author URI:        http://automattic.com/
+ * License:           GPL-2.0+
+ * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
```

Comment:
Let's add some credits / thank yous / links to the various plugins and code we've referenced, used, or drawn inspiration from.